great_expectations/expectations/core/expect_column_max_to_be_between.py

```diff
@@ -153,8 +154,9 @@ class ExpectColumnMaxToBeBetween(ColumnExpectation):
                     "estimator": "bootstrap",
                     "n_resamples": 9999,
                     "random_seed": None,
-                    "quantile_statistic_interpolation_method": "auto",
-                    "quantile_bias_std_error_ratio_threshold": 0.25,
+                    "quantile_statistic_interpolation_method": "nearest",
```

Comment:
@NathanFarmer Let us not add the setting for `quantile_statistic_interpolation_method` to be "nearest" as a constant.  It is not needed.

Reply:
This will remain `nearest` per our offline conversation.